sdk/avalon_sdk/work_order/work_order_request_validator.py

```diff
@@ -88,15 +88,17 @@ def validate_parameters(self, params):
 
         if not is_valid_hex_str(params["workerId"]):
             return False, "Invalid data format for Worker id"
-        if not is_valid_hex_str(params["workOrderId"]):
+        if not is_valid_hex_str(params["workOrderId"]) or \
+                len(params["workOrderId"]) != 64:
```

Comment:
Wherever necessary use constants.

Reply:
Added length check for params using constant in proxy model.